--- /usr/local/lib/python3.6/dist-packages/polymorph/utils.py   2020-01-21 22:29:51.000000000 +0000
+++ utils.py    2020-01-24 21:31:14.847885157 +0000
@@ -139,11 +139,7 @@
 
 
 def set_ip_forwarding(value):
-    if platform.system() == "Linux":
-        with open('/proc/sys/net/ipv4/ip_forward', 'w') as file:
-            file.write(str(value))
-            file.close()
-
+    return value
         
 def get_arpspoofer(targets, gateway, iface=None, gatewaymac=None,
           ignore=None, arpmode='rep', mac=None, ip=None):

